net-mail/cyrus-imapd/cyrus-imapd-3.2.2.ebuild

```diff
@@ -0,0 +1,208 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools flag-o-matic pam ssl-cert
+
+DESCRIPTION="The Cyrus IMAP Server"
+HOMEPAGE="https://www.cyrusimap.org/"
+SRC_URI="https://github.com/cyrusimap/${PN}/releases/download/${P}/${P}.tar.gz"
+
+LICENSE="BSD-with-attribution GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="afs backup calalarm caps clamav http kerberos ldap \
+	mysql nntp pam perl postgres replication +server sieve snmp \
+	sqlite ssl static-libs tcpd xapian"
+
+CDEPEND="
```

Comment:
Don't use `CDEPEND`, but something to describe it better. Like widely used `COMMON_DEPEND`. 
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Avoid_using_single-char_.2ADEPEND_variable